api/src/neo4j/queries/search.js

```diff
@@ -326,7 +326,7 @@ const _search = async ({
 
   // the EC field for reaction could contain ":", which is a special character
   // in this case th search term is modified to be escape and perform an exact match
```

Comment:
```suggestion
  // in this case the search term is modified to be escaped and perform an exact match
```

Reply:
Special treatment with `EC:` is not needed anymore since the symbol `:` is already sanitized. I have removed the both the code and this comment in the commit a9a85114